src/Connection.php

```diff
@@ -113,6 +113,11 @@ class Connection
      */
     protected static $multiByteSupport = null;
 
+    public static $validResources = array(
+        'socket' => true,
+        'stream' => true
+    );
+
```

Comment:
Oops :)
I just saw this commit (12 hours ago) https://github.com/mhlavac/gearman/commit/2111406cdd6128931d316370c31a478890fbb74f
And not seen this https://github.com/mhlavac/gearman/commit/037f09d657da03c7d5d3513b299b4f6a310dff8e

@mhlavac Support for PHP 5.3 really necessary?